packages/sdk/src/storage/useStorage.ts

```diff
@@ -45,11 +45,13 @@ export const useStorage = <T>(key: string, initialValue: T | (() => T)) => {
       const item = await getItem<T>(key)
 
       if (!cancel && item !== null) {
-        setData(item)
+        setTimeout(() => {
+          setData(item)
```

Comment:
can't we have racing conditions in here?

Reply:
Ain't sure, but I will remove it.